python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -398,7 +398,7 @@ def _get_state_dict(self, state):
             "total": last_op.num_outputs_total(),
             "end_time": time.time() if state != "RUNNING" else None,
             "operators": {
-                f"{op.name}{i}": {
+                f"{op.name}_{i}": {
```

Comment:
We should get someone from data team to review this.

How useful is this `i` value in the name? Is it important? Should we just remove it?

Reply:
Great question! We actually don't show it in either of the CLI or metrics, so i think we'd be safe enough to just remove it altogether.

@raulchen @bveeramani what's your take on this?

Reply:
100%, I'll follow up with @alexeykudinkin and data folks; reading the code I think one of the reason the index is required is to make the key unique (the op.name is not unique) - if that's the case, we can redesign the structure to have the op name as a field, rather than relying on the key to interpret its name

Reply:
> Great question! We actually don't show it in either of the CLI or metrics, so i think we'd be safe enough to just remove it altogether.

If it's guaranteed that the ordering of rows corresponds to the order of operators in the topology, I think it's okay to remove. (I think this currently the case)

The operator index is useful to disambiguate operators with duplicate names (e.g., multiple `MapBatches(drop_columns)`)